zerver/lib/stream_topic.py

```diff
@@ -32,5 +36,22 @@ def user_ids_muting_topic(self) -> Set[int]:
             for row in query
         }
 
+    def get_user_ids_muting_stream(self) -> Set[int]:
+        subscription_rows = Subscription.objects.filter(
+            recipient__type=Recipient.STREAM,
+            recipient__type_id=self.stream_id,
```

Comment:
Do we need to check if the subscription is active, or a former subscription, here?

Reply:
oh yes, `active` was required.
(Removed this function as this db query no longer needed)